src/plugins/wizard/public/application/components/workspace.tsx

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  EuiButton,
+  EuiContextMenu,
+  EuiContextMenuPanelItemDescriptor,
+  EuiEmptyPrompt,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiPanel,
+  EuiPopover,
+} from '@elastic/eui';
+import React, { FC, useState, useMemo, useEffect, useLayoutEffect } from 'react';
+import { useOpenSearchDashboards } from '../../../../opensearch_dashboards_react/public';
+import { IExpressionLoaderParams } from '../../../../expressions/public';
+import { WizardServices } from '../../types';
+import { validateSchemaState } from '../utils/validate_schema_state';
+import { useTypedDispatch, useTypedSelector } from '../utils/state_management';
+import { setActiveVisualization } from '../utils/state_management/visualization_slice';
```

Comment:
NIT: 
should we remove this import? since the only place uses it is a comment (toDO)

Reply:
yep. will do that :)